src/app/screens/transactionRequest/index.tsx

```diff
@@ -55,11 +55,7 @@ function TransactionRequest() {
       tokenTransferPayload.amount,
       tokenTransferPayload.memo!,
```

Comment:
Let's also add a check for the `tokenTransferPayload.memo` here to make the code safer?

Reply:
good call. it should be an optional arg in the core function

Reply:
but I don't want to refactor the core function to make it optional, because then I should also reorder the args to have memo?: string come after the required params.

maybe a compromise here is to apply the default earlier like `tokenTransferPayload.memo ?? ''`